src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,150 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+    private Node<T> newNode;
+    private Node<T> temporaryNode;
+
+    private static class Node<T> {
+        protected T element;
+        protected Node<T> prev;
+        protected Node<T> next;
+
+        protected Node(Node<T> prev, T element, Node<T> next) {
+            this.prev = prev;
+            this.element = element;
+            this.next = next;
+        }
+    }
+
+    public MyLinkedList() {
+        first = new Node<>(null, null, null);
+        last = new Node<>(null, null, null);
+    }
```

Comment:
we don't need that constructor

Reply:
i will replace this code in other part of code. without initialize have NullPoinerException whe i try to remove first element.